app/assets/stylesheets/modules/spotlight_overrides.scss

```
@@ -273,6 +273,22 @@ h5,
 .tags .nav-link {
   margin-bottom: 0.5rem;
   padding: 0.33rem 0.66rem;
+
+  &[href="/?tag=Social+justice"] {
+    color: $cardinal-red;
+    font-weight: 700;
+
+    &::before {
+      // Licensed (required to change color) from:
+      // https://thenounproject.com/search/?q=justice%20scale&i=3316424
+      content: url(social-justice.svg);
```

```suggestion
      content: image_url('social-justice.svg');
```

I /think/ we need to use `image_url` to play nice with the asset pipeline?

---

Ah, okay, thanks. It worked locally so I didn't think about it. But I pushed up an amendment.